nodejs/README.md

```diff
@@ -10,7 +10,7 @@ Specifically, these images contain everything in the [base image](../../base/REA
 - Node.js v12 (`gcr.io/distroless/nodejs:12`) and its dependencies.
 - Node.js v14 (`gcr.io/distroless/nodejs:14`) and its dependencies.
 
-**Note:** the `latest` tag maps to Node.js v10 for backwards compatibility and is now deprecated. Once Node.js v10 reaches EOL on 2021-04-30, the `latest` tag will be removed to avoid confusion. Users of these images should explicitly set the LTS version tag they wish to use.
+**Note:** the `latest` tag maps to Node.js v14 to follow the official node docker images. However we recommend that users of these images should explicitly set the LTS version tag they wish to use.
```

Comment:
I personally have no objections to having `latest` map to the latest stable node version. In the PR where I introduced node 14, there were some backwards compatibility concerns with changing `latest` away from v10.

I'd argue it's less confusing and more consistent with other docker images to make this change :)